community.md

```diff
@@ -88,6 +88,32 @@ activity is captured in email form for archival purposes.
   <li> <code>github@</code> for all activity on the GitHub repositories {% include mailing_list_links.html list="github" %} </li>
 </ul>
 
+### GitHub
+
+We use GitHub Issues as a place to report bugs, request new features, and
+track the queue of development work. For usage questions, some of our
+repositories use Discussions as an alternative to the <user@arrow.apache.org>
+mailing list {% include mailing_list_links.html list="user" %}. For those
+repositories, Discussions are mirrored to the <user@arrow.apache.org> mailing
+list {% include mailing_list_links.html list="user" %} and users are welcome to
+ask usage questions in either location. Maintainers may convert usage type
+Issues to Discussions as appropriate.
+
+Please create Issues or start Discussions on the appropriate repository:
+
+Implementations:
+
+- Go: [apache/arrow-go](http://github.com/apache/arrow-go) ([Issues](http://github.com/apache/arrow-go/issues))
+- Java: [apache/arrow-java](http://github.com/apache/arrow-java) ([Issues](http://github.com/apache/arrow-java/issues))
+- nanoarrow: [apache/arrow-nanoarrow](https://github.com/apache/arrow-nanoarrow) ([Issues](https://github.com/apache/arrow-nanoarrow/issues), [Discussions](https://github.com/apache/arrow-nanoarrow/discussions))
+- Rust: [apache/arrow-rs](http://github.com/apache/arrow-rs) ([Issues](http://github.com/apache/arrow-rs/issues), [Discussions](http://github.com/apache/arrow-rs/discussions))
+- .NET: [apache/arrow-dotnet](https://github.com/apache/arrow-dotnet) ([Issues](https://github.com/apache/arrow-dotnet/issues), [Discussions](https://github.com/apache/arrow-dotnet/discussions))
+- All Others: [apache/arrow](http://github.com/apache/arrow) ([Issues](http://github.com/apache/arrow/issues), [Discussions](http://github.com/apache/arrow/discussions)).
+
+Standards:
+
+- ADBC: [apache/arrow-adbc](https://github.com/apache/arrow-adbc) ([Issues](https://github.com/apache/arrow-adbc/issues), [Discussions](https://github.com/apache/arrow-adbc/discussions))
```

Comment:
Should we add a bullet for "all others" here under "Standards" also, pointing to the monorepo?

Reply:
Yeah I think that's good, nice catch.